spec/order_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'order.rb'
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:menu) { double() }
```

Comment:
Nice that you've started to make a double of menu but I can't see where it is used within the tests? Perhaps you were getting to this later?